packages/calcite-components/CHANGELOG.md

```diff
@@ -3,1338 +3,297 @@
-
-## [3.0.0-next.18](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-components@3.0.0-next.17...@esri/calcite-components@3.0.0-next.18) (2024-11-20)
-
-### Bug Fixes
-
-- **flow-item:** fix TS error caused by FlowItemLike type ([#10802](https://github.com/Esri/calcite-design-system/issues/10802)) ([22c46c7](https://github.com/Esri/calcite-design-system/commit/22c46c7178ce0f97994d4302f8c164f9e928302b)), closes [#10801](https://github.com/Esri/calcite-design-system/issues/10801) [#10482](https://github.com/Esri/calcite-design-system/issues/10482)
-- **t9n:** make TypeScript inline .json type imports ([#10804](https://github.com/Esri/calcite-design-system/issues/10804)) ([5ad6784](https://github.com/Esri/calcite-design-system/commit/5ad6784f7ba5a2e8a083e00c45d48d476e5f4cf9)), closes [#10803](https://github.com/Esri/calcite-design-system/issues/10803)
-
-## [3.0.0-next.17](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-components@3.0.0-next.16...@esri/calcite-components@3.0.0-next.17) (2024-11-20)
+## [3.0.0](https://github.com/Esri/calcite-design-system/compare/@esri/calcite-components@2.13.2...@esri/calcite-components@3.0.0) (2025-02-08)
 
 ### ⚠ BREAKING CHANGES
 
+- **list, list-item, list-item-group, filter:** The spacing changes affect content in the `actions-start` slot in addition to other rendered elements. Some styling updates may need to be applied to slotted actions or other adjustments based on the new scales.
+- **tokens:** Size and Space token values have changed. Line Height tokens exported for JS and ES6 are now unitless.
+- **color-picker:** The default width and height of the `color-picker` have been updated. Developers should review and adjust their layouts to accommodate these changes.
+- **list:** Choose a `displayMode` of "nested" or "flat" according to space and nesting requirements.
+- **chip:** The `label` property is now required and `value` is optional.
+- **radio-button:** Removes the deprecated property `guid` from `calcite-radio-button`.
+- **progress:** Refactors `progress` to use the value range of 0-100. Developers will need to provide a function to map existing values or use the updated range.
+- **shell-panel:** Removes the following deprecated properties from `calcite-shell-panel`: `detachedHeightScale`, `detached` and the `--calcite-shell-panel-detached-max-height` CSS property.
+- **color-picker:** Removes the following deprecated properties from `calcite-color-picker`: `hideChannels`, `hideHex` and `hideSaved`.
+- **action-group:** Removes the deprecated `layout` property by converting it to an internal property.
 - **table:** Removes the deprecated property `zebra` from `calcite-table`.
-
-Developers will need to replace `calcite-table`'s "zebra" property with "striped".
-
 - **checkbox:** Removes the deprecated `guid` property from `calcite-checkbox`.
-
-Developers may need to replace `calcite-checkbox`'s "guid" property with a unique "id" property.
```

Comment:
These "Developers will need" and "Developers may need" lines did not make it into the release-please changelog for v3. Please create a followup PR if necessary